docs/layout/grid.md

```diff
@@ -430,3 +430,45 @@ Easily change the order of our built-in grid columns with `.col-md-push-*` and `
 </div>
 {% endexample %}
 </div>
+
+## Predefined Flexbox classes
+
+When you compile [Bootstrap with Flexbox support](../getting-started/flexbox/) you can use some extra predefined grid classes.
+
+### Example: Alignment for every column in row
+
+You can set the vertical alignment of every column in a row with the `row-*-top`, `row-*-center` and `row-*-bottom` classes. These classes are avaible for each grid.
```

Comment:
Use a leading dot when mentioning a class name.
E.g. Use `.row-*-center`, not `row-*-center`